src/vma/dev/ring_slave.cpp

```diff
@@ -222,6 +224,22 @@ void ring_slave::flow_tcp_del_all()
 	}
 }
 
+bool ring_slave::request_more_buffers(descq_t &local_pool, uint32_t count, uint32_t lkey, bool is_rx)
+{
+	buffer_pool *global_pool = is_rx ? g_buffer_pool_rx : g_buffer_pool_tx;
+
+	ring_logfuncall("Allocating additional %d buffers for internal use", count);
+
+	bool res = global_pool->get_buffers_thread_safe(local_pool,
+			this, count, lkey);
+	if (!res) {
+		ring_logfunc("Out of mem_buf_desc from %s free pool for internal object pool", is_rx ? "rx" : "tx");
+		return false;
+	}
+
```

Comment:
Is statistic missed?
https://github.com/Mellanox/libvma/pull/695/commits/ee02fea91aaaddee4a43a91c6bd80e2318c6e7a9#diff-1e15a9c45d43c8e00766447137365b1fL335

Reply:
no, it is updated from compensate_qp_poll_success. from example :
https://github.com/Mellanox/libvma/commit/ee02fea91aaaddee4a43a91c6bd80e2318c6e7a9#diff-7d5519a685a26b0a2b28191d5eac7783L64